src/Lean/Elab/Command.lean

```diff
@@ -612,6 +650,9 @@ Interrupt and abort exceptions are caught but not logged.
 private def liftAttrM {α} (x : AttrM α) : CommandElabM α := do
   liftCoreM x
 
+/-- Return the stack of all currently active scopes:
+the base scope always comes last; new scopes are prepended in the front.
+In particular, the current scope is always the first element. -/
```

Comment:
For one sentence docstrings you can put them on one line, but the style is that multiline ones have `/--` and `-/` on their own lines:
```suggestion
/--
Return the stack of all currently active scopes:
the base scope always comes last; new scopes are prepended in the front.
In particular, the current scope is always the first element.
-/
```
This holds for field docstrings as well. Would you mind updating `Scope`?

Reply:
Sure, done. Do you prefer `levelNames` to also be put on separate lines (for me, that's morally "one sentence")?

Reply:
For `levelNames`, your options are
```
  /-- The current list of names for universe level variables to use for new declarations. This is managed by the `universe` command. -/
  levelNames    : List Name := []
```
or
```
  /--
  The current list of names for universe level variables to use for new declarations.
  This is managed by the `universe` command.
  -/
  levelNames    : List Name := []
```
(Along with some minor variations.) I'll leave the choice to you.

I shouldn't have said "one sentence" but rather "one line".

Reply:
Changed, to the first version.